src/sentry/static/sentry/app/views/organizationIntegrations/integrationRepos.jsx

```diff
@@ -184,8 +192,10 @@ export default class IntegrationRepos extends AsyncComponent {
         </DropdownButton>
       );
     }
-    const repositories = new Set(this.state.itemList.map(i => i.name));
-    const repositoryOptions = (this.state.integrationRepos.repos || []).filter(
+    const repositories = new Set(
+      this.state.itemList.filter(item => item.integrationId !== 'None').map(i => i.name)
```

Comment:
Why is it the string `None`?

Reply:
This seems to be how it comes back when the integrationId isn't set

Reply:
Can we figure this out? The backend should be returning null. cc @mnoble